styx-scheduler-service/src/main/java/com/spotify/styx/BackfillTriggerManager.java

```diff
@@ -135,85 +139,95 @@ void tick0() {
   private void triggerAndProgress(Backfill backfill) {
     // Do not include all backfill spans in parent tick span to avoid it growing too big
     tracer.spanBuilderWithExplicitParent("Styx.BackfillTriggerManager.triggerAndProgress", null)
-        .startSpanAndRun(() -> triggerAndProgress0(backfill));
+        .startSpanAndRun(() -> Try.run(() -> triggerAndProgress0(backfill)));
```

Comment:
It looks to me like Try.run will swallow unchecked exceptions without logging them. How about just using `guard` like above, which will log any uncaught exceptions?

Reply:
`guard` doesn't allow checked exceptions

Reply:
Now propagating the exception though